route: The config route should only include static routes and boot routes

cathay4t opened the pull request:
The ip command created route is boot protocol, we should include it
in route.config also.

Both python and rust code are updated.

ffmancera reviewed (dismissed):
LGTM!

ffmancera reviewed (changes requested):
Please, check my comment about: https://github.com/nmstate/nmstate/runs/4614122720?check_suite_focus=true#step:5:2764

ffmancera commented:
CI is failing on a route test that was not failing before, could you take a look please? https://github.com/nmstate/nmstate/runs/4614122720?check_suite_focus=true#step:5:2764

cathay4t committed:
route: The config route should only include static and boot routes

The ip command created route is boot protocol, we should include it
in route.config also.

Both python and rust code are updated.

Signed-off-by: Gris Ge <fge@redhat.com>

cathay4t commented:
CI fixed.

ffmancera reviewed (approved):
LGTM!
